src/web/mjs/connectors/templates/SpeedBinb.mjs

```diff
@@ -20,7 +20,11 @@ export default class SpeedBinb extends Connector {
                 data = data[0];
                 if( data.dataset['ptbinb'] && data.dataset['ptbinbCid'] )
                 {
-                    return this._getPageList_v016113( chapter.id + '&cid=' + data.dataset['ptbinbCid'], data.dataset.ptbinb );
```

Comment:
We should use URL object to handle URIs instead of dealing with manual string composition.
```javascript
let uri = new URL(chapter.id, this.url);
uri.searchParams.set('cid', data.dataset['ptbinbCid']);
return this._getPageList_v016113(uri.pathname + uri.search, data.dataset.ptbinb);
```